package.json

```diff
@@ -45,7 +45,7 @@
 		"@types/qrcode-svg": "1.1.1",
 		"@types/systemjs": "6.1.1",
 		"@types/winreg": "1.2.31",
-		"better-sqlite3": "git+https://github.com/tutao/better-sqlite3-sqlcipher#c6ee076a4f15671d8a0fa7ddc16d12b179deb350",
+		"better-sqlite3": "git+https://github.com/tutao/better-sqlite3-sqlcipher#7e6d8b36cd804b4d5d07264c9a2f4a8820881586",
```

Comment:
why did this one change?

Reply:
Rebase https://github.com/tutao/tutanota/pull/6842